clients/web/src/templates/body/systemConfiguration.jade

```diff
@@ -46,7 +74,7 @@ form.g-settings-form(role="form")
       option(value="yesmod", selected=(settings['core.add_to_group_policy'] ==
              "yesmod")) Yes by group administrators and moderators unless disabled per group
 
-  .g-collection-create-policy-container
+  .g-settings-form-container
```

Comment:
@zachmullen Since this class name is so specific to the one place that it was used before, I'd still like to copy the content to a new "g-settings-form-container", and provide a deprecated alias (with @extend) named "g-collection-create-policy-container". That should encourage more reuse of this class as a useful grouping mechanism while maintaining compatibility.